tests/core_tests/rct.cpp

```diff
@@ -109,12 +109,12 @@ bool gen_rct_tx_validation_base::generate_with(std::vector<test_event_entry>& ev
     //fill outputs entry
     tx_destination_entry td;
     td.addr = miner_accounts[n].get_keys().m_account_address;
-    td.amount = 7390000000000;
+    td.amount = 17390000000000;
     std::vector<tx_destination_entry> destinations;
     destinations.push_back(td);
     destinations.push_back(td);
     destinations.push_back(td);
-    destinations.push_back(td); // 30 -> 7.39 * 4
+    destinations.push_back(td); // 70 -> 17.39 * 4
```

Comment:
Out of curiosity, I'm guessing something broke with those initial values, what was it ?

Reply:
The coinbase on V2 fork doubled due to the blocktime changing from 120s to 240s, so the numbers needed to be adjusted.